webapp/src/js/bootstrapper/index.js

```diff
@@ -128,7 +128,8 @@
   module.exports = function(POUCHDB_OPTIONS, callback) {
     var dbInfo = getDbInfo();
     var userCtx = getUserCtx();
-    if (!userCtx) {
+    var hasForceLoginCookie = document.cookie.indexOf('login=force') > -1;
```

Comment:
`document.cookie.includes('login=force')`?

Reply:
`const` is so much cooler than `var`. It can lead to compile-time performance optimizations. It is block scoped instead of function scoped. It has better readability semantics...